config.go

```diff
@@ -94,6 +94,7 @@ type SessionConfig struct {
 	SessionCookieLifeTime   int
 	SessionAutoSetCookie    bool
 	SessionDomain           string
+	DisableHTTPOnly         bool // used to allow for cross domain cookies/javascript cookies.
```

Comment:
Would SessionDisableHTTPOnly work? I think that the field should be Disable rather than Enable, because if something happens and the field isn't set, booleans default to false.